time_stamper_cv/include/configuration.h

```diff
@@ -35,11 +35,16 @@ struct LedRowConfig {
  * Parameters for the OpenCV SimpleBlobDetector. See OpenCV docs for more details.
  *
  * @param led_row_config
- * See struct above for more details.
+ * LedRowConfig with reference key.
+ * See @struct LedRowConfig above for more details.
  */
+typedef std::map<std::string, LedRowConfig> LedRowConfigRepository;
 
 struct ImageProcessorConfig {
   int tolerance{10};
   cv::SimpleBlobDetector::Params params;
-  LedRowConfig led_row_config{{0, 0}, {6, 0}, 16};
+  LedRowConfigRepository led_row_config {
+      {"BottomRow", {{0, 0}, {6, 0}, 16}},
+      {"TopRow", {{0, 17}, {6, 0}, 10}}
```

Comment:
In case this code is still up for debate or as a recommendation for the future: 
Try to avoid repeating string literals in your code base. Consider `enum class` types instead or, if string literals are also printed, think of having `const char *` constants that you can use. Both ways the compiler can detect typos or similar mistakes which it can't in you current approach.